ZipBuilder/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -23,12 +23,12 @@ private enum Architecture: String, CaseIterable {
     case device = "iphoneos"
     case simulator = "iphonesimulator"
 
-    /// Arguments that should be included as part of the build process for each target platform.
-    func extraArguments() -> [String] {
```

Comment:
Is there a reason for calling it "extra" instead of "other"?

Reply:
Nope, no particular reason. I can change to other.